projects/addon-editor/components/toolbar/toolbar.component.ts

```diff
@@ -68,7 +72,7 @@ export class TuiToolbarComponent {
     readonly texClicked = new EventEmitter<void>();
 
     @Output()
-    readonly attachClicked = new EventEmitter<void>();
+    readonly attachClicked = new EventEmitter<TuiEditorAttachedFile[]>();
```

Comment:
Seem like it's no longer `attachClicked` but rather `filesAttached`. I think original name suggested the actual attach code would be handled externally. For example, if people would want to show some progress or anything like that. I know it's possible with current approach as well. Maybe we can later add an example where `fileLoader` would show an alert with progress bar?

Reply:
renamed to `filesAttached`